File: scala-gatling/src/gatling/scala/org/openapitools/client/model/ResponseMapInfo.scala

package org.openapitools.client.model


case class ResponseMapInfo (
    _maps: List[ResponseMapInfoMap]
)
object ResponseMapInfo {
    def toStringBody(var_maps: Object) =
        s"""
        | {
        | "maps":$var_maps
        | }
        """.stripMargin
}

File: kotlin-spring/src/main/kotlin/org/openapitools/model/ResponseMapInfo.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import org.openapitools.model.ResponseMapInfoMap
import javax.validation.constraints.DecimalMax
import javax.validation.constraints.DecimalMin
import javax.validation.constraints.Max
import javax.validation.constraints.Min
import javax.validation.constraints.NotNull
import javax.validation.constraints.Pattern
import javax.validation.constraints.Size
import javax.validation.Valid

/**
 * 
 * @param maps 
 */
data class ResponseMapInfo(

    @field:Valid
    @field:JsonProperty("maps", required = true) val maps: kotlin.collections.List<ResponseMapInfoMap>
) {

}

